Lesson11/src/main/java/task4/Main.java

```diff
@@ -0,0 +1,22 @@
+package task4;
+
+public class Main {
+    public static void main(String[] args) {
+        Apple apple = new Apple(7);
+        System.out.println("Цена за " + apple.getWeight() + " кг яблок - " + apple.fruitCost() + " рублей");
+
+        Apricot apricot = new Apricot(4.56);
+        System.out.println("Цена за " + apricot.getWeight() + " кг груш - " + apricot.fruitCost() + " рублей");
+
+        Pear pear = new Pear(11.3);
+        System.out.println("Цена за " + pear.getWeight() + " кг груш - " + pear.fruitCost() + " рублей");
+
+        Fruit[] fruits = new Fruit[]{apple, apricot};
+
+        FructShop fructShop = new FructShop();
+        fructShop.priceFruct(fruits);
+
+        // как округлить double до сотых?)
```

Comment:
https://www.google.com/search?q=java+%D0%BA%D0%B0%D0%BA+%D0%BE%D0%BA%D1%80%D1%83%D0%B3%D0%BB%D0%B8%D1%82%D1%8C+double+%D0%B4%D0%BE+%D1%81%D0%BE%D1%82%D1%8B%D1%85%3F)&oq=java+%D0%BA%D0%B0%D0%BA+%D0%BE%D0%BA%D1%80%D1%83%D0%B3%D0%BB%D0%B8%D1%82%D1%8C+double+%D0%B4%D0%BE+%D1%81%D0%BE%D1%82%D1%8B%D1%85%3F)&aqs=chrome..69i57.1574j0j15&sourceid=chrome&ie=UTF-8